test/test-apps3buckets-create.js

```diff
@@ -1,55 +1,46 @@
-"use strict";
-
-const assert = require('chai').assert;
+const { assert } = require('chai');
 const nock = require('nock');
 
 const config = require('../app/config');
 const handlers = require('../app/apps3buckets/handlers');
-const url_for = require('../app/routes').url_for;
+const { url_for } = require('../app/routes');
 
 
 describe('Edit bucket form', () => {
-
   describe('when granting access to user', () => {
-
     it('make request to API', () => {
       const bucket_id = 42;
       const app_id = 123;
 
       // Mock `POST /apps3buckets` request
       const post_apps3buckets = nock(config.api.base_url)
-        .post(`/apps3buckets/`, {
+        .post('/apps3buckets/', {
           app: app_id,
           s3bucket: bucket_id,
-          access_level: "readonly",
+          access_level: 'readonly',
         })
         .reply(201);
 
-      let request = new Promise((resolve, reject) => {
-        let req = {
-          body: {
-            app_id: app_id,
-            bucket_id: bucket_id,
-          },
+      const request = new Promise((resolve, reject) => {
+        const req = {
+          body: { app_id, bucket_id },
         };
-        let res = {
+        const res = {
           redirect: (redirect_url) => {
             resolve(redirect_url);
-          }
+          },
         };
 
         handlers.create[1](req, res, reject);
       });
 
       return request
         .then((redirect_url) => {
-          assert(post_apps3buckets.isDone(), `Make POST request to /apps3buckets (API)`);
+          assert(post_apps3buckets.isDone(), 'Make POST request to /apps3buckets (API)');
```

Comment:
The string parameter is the error message shown if the assertion fails. It should explain why the assertion failed, eg: "Did not make POST request to API endpoint /apps3buckets"

Reply:
👍